message_ix_models/data/node/ZMB.yaml

```diff
@@ -0,0 +1,9 @@
+# Codes for the "node" dimension of the MESSAGE-IL model
+
+World:
+  name: World
+  description: MESSAGE-ZA regions
```

Comment:
I would strongly suggest to use either “MESSAGE[ix]-Zambia” or “MESSAGE[ix]-ZM,” because [ZA is the ISO 3166-1 alpha-2 code for South Africa—**not** Zambia](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#ZA).